common/djangoapps/external_auth/views.py

```diff
@@ -263,75 +259,6 @@ def _flatten_to_ascii(txt):
         return unicode(unicodedata.normalize('NFKD', txt).encode('ASCII', 'ignore'))
 
 
-@ensure_csrf_cookie
-def _signup(request, eamap, retfun=None):
```

Comment:
@waheed if necessary, it looks like you could replace calls to `_signup` with calls to [`login_and_registration_form`](https://github.com/edx/edx-platform/pull/11549/files#diff-7960baf09ff3f493a2b3fedefa7aad05R45). However, it's not clear to me why you need to do that. This function appears to have been ported: https://github.com/edx/edx-platform/blob/master/common/djangoapps/external_auth/login_and_register.py#L77.